tcms/kiwi_auth/tests/test_forms.py

```diff
@@ -0,0 +1,68 @@
+import importlib
+
+from django.conf import settings
+from django.test import TestCase, override_settings
+
+from tcms.kiwi_auth import forms
+
+
+class TestRecaptchaField(TestCase):
+    def setUp(self):
+        self.data = {
+            "username": "test_user",
+            "password1": "password",
+            "password2": "password",
+            "email": "new-tester@example.com",
+        }
+
+    @override_settings(INSTALLED_APPS=settings.INSTALLED_APPS + ["captcha"])
+    def test_captcha_required_when_enabled(self):
+        importlib.reload(forms)
+
+        RegistrationForm = forms.RegistrationForm
+
+        form = RegistrationForm(data=self.data)
+
+        self.assertFalse(form.is_valid())
+
+        self.assertIn("captcha", form.errors.keys())
+
+        self.assertIn("This field is required.", form.errors["captcha"])
```

Comment:
Error messages are translated, that's why some of the CI jobs fail.

Reply:
Changed it